### PR TITLE
FSF- Added brew cask install multipass

### LINKED ISSUE #2049: FSF - Add brew cask install multipass
Add brew method of installing multipass to remain consistent with the installation of snapcraft. It also means that the user does not have to navigate away from the FSF tutorial.

Also, change the order of installation steps and add link to [documentation](https://docs.snapcraft.io/snapcraft-overview) for  ‘Install Snapcraft’, see design [here](https://zpl.io/aBKMq3O).

### PR DESCRIPTION
**Pull request by hughesjordan**
Tiny first PR..

Fixes #2049
 
### QA Steps: 
1. Pull the branch or run the demo service
2. Go to the FSF and go to the 'Install Snapcraft' stage of the tutorial
3. The install steps should be re-ordered Homebrew, Snapcraft, Multipass
4. You should be able to `brew cask install Multipass`

### CODE REVIEW

**Comment by therealjuan** [comment]
Just added a 🚧emoji, removed Review labels and added Don't merge label so everyone is aware that is not something to merge for now.

**Comment by hughesjordan** [comment]
Another issue had already been created for the other tiny changes #2048

**Comment by Lukewh** [comment]
Doesn't seem to have anything about `brew cask install` added? Just has a link to snapcraft docs (https://docs.snapcraft.io/snapcraft-overview) added to the last step.
The order is also not as in the QA steps: Multipass, Homebrew, Snapcraft

**Comment by hughesjordan** [comment]
@Lukewh when I use ./run I have the following:



I've no idea what is going on tbh. I'm on my branch, I re added and committed? Perhaps done something wrong in setting up the PR

File: templates/first-snap/_install-macos.html [hunk 1]
```diff
@@ -12,24 +12,27 @@ <h4>How would you like to set up your build environment?</h4>
       <ol class="p-list-step">
         <li class="p-list-step__item">
             <h4 class="p-list-step__title">
-              <span class="p-list-step__bullet">1</span>
-              <a href="https://multipass.run/install" target="_blank" class="p-link--external">Install Multipass</a>, a virtualisation product from <br/>Canonical that can set up a complete Linux environment <br/>in a couple of minutes
+              <span class="p-list-step__bullet">2</span>
```
Comment (Lukewh) [inline templates/first-snap/_install-macos.html]:
```suggestion
              <span class="p-list-step__bullet">1</span>
```

File: templates/first-snap/_install-macos.html [hunk 2]
```diff
@@ -12,24 +12,27 @@ <h4>How would you like to set up your build environment?</h4>
...
         <li class="p-list-step__item">
-          <h4 class="p-list-step__title">
-            <span class="p-list-step__bullet">3</span>
-            Install Snapcraft, the command-line tool for building snaps
-          </h4>
-          {% set snippet_value = "brew install snapcraft" %}
-          {% set snippet_id = "macos-install-snapcraft" %}
-          {% include "/partials/_code-snippet.html" %}
+            <h4 class="p-list-step__title">
+              <span class="p-list-step__bullet">3</span>
```
Comment (Lukewh) [inline templates/first-snap/_install-macos.html]:
```suggestion
              <span class="p-list-step__bullet">2</span>
```

File: templates/first-snap/_install-macos.html [hunk 3]
```diff
@@ -12,24 +12,27 @@ <h4>How would you like to set up your build environment?</h4>
...
+              <a href="https://docs.snapcraft.io/snapcraft-overview" target="_blank" class="p-link--external">Install Snapcraft</a>, the command-line tool for building snaps:
+            </h4>
+            {% set snippet_value = "brew install snapcraft" %}
+            {% set snippet_id = "macos-install-snapcraft" %}
+            {% include "/partials/_code-snippet.html" %}
+        </li>
+        <li class="p-list-step__item"> 
+            <h4 class="p-list-step__title">
+              <span class="p-list-step__bullet">1</span>
```
Comment (Lukewh) [inline templates/first-snap/_install-macos.html]:
```suggestion
              <span class="p-list-step__bullet">3</span>
```

**Pull request merged**